### ISSUE TITLE
Pre-training model import

### ISSUE THREAD

**Issue by czy112** [megvii-model/ShuffleNet-Series#37]
address = "E:/ShuffleNetV2.1.0x.pth.tar"
  pretrained_state_dict = torch.load(address)
  self.load_state_dict(pretrained_state_dict, strict=False)

When I imported in this way, I found that the loss of model training changed slowly, almost as much as when the pre-training model was not loaded
Do you know the reason? Thank you very much

**Comment by nmaac** [comment]
Hi @czy112 ,

Please follow line 163 in train.py to load the model. The fine-tune training loss would not change faster than that from scratch.

**Comment by czy112** [comment]
i follow your suggest:
    address = "E:/shufflenetcenter/ShuffleNetV2+.Small.pth.tar"
    pretrained_state_dict = torch.load(address)
    model.init_weights(pretrained=True)
    print( model.load_state_dict(pretrained_state_dict['state_dict'], strict=False))
but i missed this error:
Missing key(s) in state_dict: "first_conv.0.weight", "first_conv.1.weight", "first_conv.1.bias", "first_conv.1.running_mean", "first_conv.1.running_var",
can you help me?
thanks

**Comment by nmaac** [comment]
The error is caused by incorrect nn.DataParallel model loading, please take a look at [pytorch tutorial](https://pytorch.org/tutorials/beginner/saving_loading_models.html).
